src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -494,42 +493,13 @@ def test_ad_invalid_iteration_space():
             "'colours'." in str(excinfo.value))
 
 
-def test_arg_descriptor_invalid_fs1():
```

Comment:
Presumably these tests have been moved into the new test file?

Reply:
`test_arg_descriptor_invalid_fs1` is now called `test_arg_descriptor_invalid_fs` and is located in `lfric_field_test.py`. It tests bth real and integer field arguments for invalid function space. `test_arg_descriptor_invalid_fs2`  is now `test_ad_op_type_5th_arg_not_space` in `dynamo0p3_lma_test.py` (operator test originally).